elements/rh-spinner/docs/10-style.md

```diff
@@ -30,7 +30,7 @@ A spinner comes in large, medium, and small sizes. Each size includes an optiona
 <br>
 
 | Size | Element | Current value |
-| ---- | ------- | ------------- |
+| ------- | ------- | ------- |
```

Comment:
if you want to automatically format these, you might like 
https://learn.microsoft.com/en-us/contribute/docs-authoring/reformat-table
or 
https://marketplace.visualstudio.com/items?itemName=pharndt.vscode-markdown-table